### PR TITLE
Correction: Changed "नमस्कार l" to "नमस्कार."  #29

### PR DESCRIPTION
**Pull request by SarveshDevrukhkar**
@paramsiddharth Issue ID #29

As per I mentioned I have made changes in the file, so please review it. 

Thank You! :smiley:

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -55,4 +55,4 @@ export const SITE_URL = SITE_URL_;
 export const PAGE_SIZE = PAGE_SIZE_;
 export const DATE_FORMAT = DATE_FORMAT_;
 export const PUBLIC_DATE_FORMAT = PUBLIC_DATE_FORMAT_;
-export const UTC_OFFSET = UTC_OFFSET_;
\ No newline at end of file
+export const UTC_OFFSET = UTC_OFFSET_;
```
Comment (paramsiddharth) [inline src/constants.ts:58]: This change is unwanted — The style guide rules require files to not end with a new line. Could you please revert this change?

**Review by paramsiddharth — CHANGES_REQUESTED** [review]
@SAD0XER Hi! 😊 Really appreciate the contribution. Just one little change and we'll be good to merge!

File: src/constants.ts (hunk 1 above)
Comment (SarveshDevrukhkar) [reply src/constants.ts:58]: I understand that.
Only just because of this thing, previously I close the request!

I don't know why that line gets changed automatically. 
Actually, I change only the wrong part of the code. But I don't know what happens to that end of the line. I don't even touch that line, that last line change is unexpected and automatic.

**So, what should I do?
Should I create a new branch with the correction?  or revert this change?**

Comment (SarveshDevrukhkar) [reply src/constants.ts:58]: **Before**
```diff
- export const UTC_OFFSET = UTC_OFFSET_;
```
**After**
```diff
+ export const UTC_OFFSET = UTC_OFFSET_;
```

And If you observe the code diff, the previous line and current line are the same.

Comment (paramsiddharth) [reply src/constants.ts:58]: > I don't know why that line gets changed automatically. Actually, I change only the wrong part of the code. But I don't know what happens to that end of the line. I don't even touch that line, that last line change is unexpected and automatic.

It is a change made by your code editor. :) You can disable it by changing auto-prettifying settings in your editor.

Comment (SarveshDevrukhkar) [reply src/constants.ts:58]: Actually, that time I made changes on github, not on any code editor.🤷🏻‍♂️
I don't know why this happens on github.

Then next time new PR was edited on vs code, there everything was smooth, without any problem.
But anyways, I will see that setting later :)
